### ISSUE TITLE
Fatal exit when opening wallet

### ISSUE THREAD

**Issue by mverrilli** [decred/decrediton#1658]
After installed (and even uninstalling/reinstalling) Upon opening the wallet, it caused a crash without any kind of error popup. So I tried deleting the wallet and restoring from seed and had the same issue. Nothing seems interesting in the wallet log or decrediton.log.

If running from a cmd prompt, the following was dumped out. 

```
C:\Users\xxxxxxx\AppData\Local\Programs\decrediton\resources\app.asar\main.js:8
!function(){function r(e){var t=!1;return function(){if(t)throw new Error("Callback was already called.");t=!0,e.apply(a,arguments)}}var a,i,s={},c=function(){};a="object"==typeof window&&this===window?window:"object"==typeof global&&this===global?global:this,null!=a&&(i=a.async),s.noConflict=function(){return a.async=i,s};var u,l=Object.prototype.toString,f=Array.isArray||function(e){return"[object Array]"===l.call(e)},p=function(e,t){for(var r=-1,n=e.length;++r<n;)t(e[r],r,e)},d=function(e,t){for(var r=-1,n=e.length,o=Array(n);++r<n;)o[r]=t(e[r],r,e);return o},h=function(e,t,r){return p(e,function(e,n,o){r=t(r,e,n,o)}),r},m=function(e,t){p(g(e),function(r){t(e[r],r)})},g=Object.keys||function(e){var t=[];for(var r in e)e.hasOwnProperty(r)&&t.push(r);return t},v=function(e,t){t=t||0;var r=-1,n=e.length;t&&(n-=t,n=n<0?0:n);for(var o=Array(n);++r<n;)o[r]=e[r+t];return o};"function"==typeof setImmediate&&(u=setImmediate),"undefined"!

TypeError: a.sendSync is not a function
    at ChildProcess.<anonymous> (C:\Users\xxxxxxx\AppData\Local\Programs\decrediton\resources\app.asar\main.js:1:63674)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)

```

**Comment by matheusd** [comment]
This usually a symptom of some earlier problem (dcrd/dcrwallet not starting, etc).

Please try running decrediton from the command line with --debug or post the most recent lines in your decrediton.log.

**Comment by alexlyp** [comment]
https://github.com/decred/dcrwallet/pull/1279

So this has been fixed in dcrwallet, but unfortunately not included in the release.  I'll update you here no our plans to get this fixed.

**Comment by alexlyp** [comment]
Fixed in v1.3.1 release
